barretenberg/cpp/src/barretenberg/dsl/acir_format/witness_constant.cpp

```diff
@@ -12,10 +12,17 @@ bb::stdlib::cycle_group<Builder> to_grumpkin_point(const WitnessOrConstant<FF>&
                                                    bool has_valid_witness_assignments,
                                                    Builder& builder)
 {
-    using bool_ct = bb::stdlib::bool_t<Builder>;
     auto point_x = to_field_ct(input_x, builder);
     auto point_y = to_field_ct(input_y, builder);
-    auto infinite = bool_ct(to_field_ct(input_infinite, builder));
+    // We assume input_infinite is boolean, so we do not add a boolean gate
```

Comment:
pedantic point: if `infinite` is defined as a `bool_ct` then we don't need to assume `input_infinite` is boolean - the `bool_ct` constructor will explicitly add a boolean gate.